.github/CODEOWNERS

```diff
@@ -86,7 +86,8 @@ exporter/zipkinexporter/                                 @open-telemetry/collect
 extension/asapauthextension/                             @open-telemetry/collector-contrib-approvers @jamesmoessis @MovieStoreGuy
 extension/awsproxy/                                      @open-telemetry/collector-contrib-approvers @Aneurysm9 @mxiamxia
 extension/basicauthextension/                            @open-telemetry/collector-contrib-approvers @jpkrohling @svrakitin
-extension/bearertokenauthextension/                      @open-telemetry/collector-contrib-approvers @jpkrohling @pavankrish123 @frzifus
+extension/bearertokenauthextension/                      @open-telemetry/collector-contrib-approvers @jpkrohling @pavankrish123
```

Comment:
Removal of @frzifus looks unrelated to this PR. If it's necessary, let's do it in a separate PR?

Reply:
That is my mistake trying to rebase